shell/platform/windows/flutter_windows_view.cc

```diff
@@ -161,11 +168,11 @@ void FlutterWindowsView::SetEventPhaseFromCursorButtonState(
   // For details about this logic, see FlutterPointerPhase in the embedder.h
   // file.
   event_data->phase =
-      mouse_state_.buttons == 0
-          ? mouse_state_.flutter_state_is_down ? FlutterPointerPhase::kUp
-                                               : FlutterPointerPhase::kHover
-          : mouse_state_.flutter_state_is_down ? FlutterPointerPhase::kMove
-                                               : FlutterPointerPhase::kDown;
+      mouse_state_.buttons == 0            ? mouse_state_.flutter_state_is_down
+                                                 ? FlutterPointerPhase::kUp
+                                                 : FlutterPointerPhase::kHover
+      : mouse_state_.flutter_state_is_down ? FlutterPointerPhase::kMove
+                                           : FlutterPointerPhase::kDown;
```

Comment:
This is particularly grating to look at, but I guess a product of the clang 12 roll. :(